packages/flutter_tools/test/general.shard/fuchsia/fuchsia_device_test.dart

```diff
@@ -458,12 +457,12 @@ void main() {
         () => device.takeScreenshot(globals.fs.file('file.invalid')),
         throwsA(equals('file.invalid must be a .ppm file')),
       );
-    }, testOn: 'posix');
+    });
```

Comment:
can this be `testWithoutContext`?

Reply:
The fuchsia device still uses context :(

Reply:
LGTM then